demisto_sdk/commands/common/tests/integration_test.py

```diff
@@ -1,4 +1,5 @@
 import pytest
+from yaml import safe_load
```

Comment:
import the module in order to give context to safe_load function (using `yaml.safe_load` explain that you are loading a yaml file)
```suggestion
import yaml
```